apps/web/components/ui/MediaCard/MediaCard.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div v-if="showComponent" :class="['flex flex-col md:flex-row items-center', positionClass]">
     <div v-if="image" :class="['w-full', 'md:w-1/2', { 'mb-4': text }]">
-      <img :src="image" alt="Media Image" class="w-full h-auto object-cover" />
+      <img :src="image" alt="Media Image" width="728" height="485" class="w-full h-auto object-cover" />
```

Comment:
**question**: Should we use static values or `max-` ones? If someone uploaded a 700x300 image, it'd be stretched with this setup, right?

Reply:
You can't use max-width as an attribute in html, max-width can be only set in css. To answer your question, it will be defaulted to  width="728" height="485"